Jenkinsfile

```diff
@@ -264,7 +264,13 @@ def bazelTest = {
                             if (env.BRANCH_NAME != 'master') {
                                 cmd += ' --remote_upload_local_results=false';
                             }
-                            sh label: 'Bazel Build', script: cmd + ' --google_credentials=$GCS_SA_KEY ...';
+                            cmd += ' --google_credentials=$GCS_SA_KEY';
+                            try {
+                                sh label: 'Bazel Build', script: cmd + ' ...';
+                            } catch (e) {
+                                currentBuild.result = 'FAILURE';
```

Comment:
Nice!

Just some thoughts, I'm not suggesting making any changes:

I think `currentBuild.result = 'FAILURE';` is an interesting touch. Consider network outage. We really only want to try again if there were build and not transient errors as continuing makes transient errors less obvious. 

Does the exception contain any useful information? The catch is very broad, can or should the catch be narrowed to catch more specific errors only?